wgengine/magicsock/magicsock.go

```diff
@@ -1206,13 +1207,15 @@ func (c *Conn) sendUDP(ipp netaddr.IPPort, b []byte) (sent bool, err error) {
 	return
 }
 
+var errEPERM error = syscall.EPERM
+
 // sendUDP sends UDP packet b to addr.
 // See sendAddr's docs on the return value meanings.
 func (c *Conn) sendUDPStd(addr *net.UDPAddr, b []byte) (sent bool, err error) {
 	switch {
 	case addr.IP.To4() != nil:
 		_, err = c.pconn4.WriteTo(b, addr)
-		if err != nil && c.noV4.Get() {
+		if err != nil && (c.noV4.Get() || errors.Is(err, errEPERM)) {
```

Comment:
Something somewhere deserves a comment about why we ignore permanent errors

Reply:
There’s also some net package abstract layer for this right? The comment should probably mention why we don’t use it

Reply:
PTAL